src/askem_beaker/contexts/mira_model_edit/context.py

```diff
@@ -94,7 +94,7 @@ async def send_mira_preview_message(
 			raise
 
 	@intercept()
-	async def reset_request(self, message):
+	async def reset_mira_request(self, message):
```

Comment:
Just as a heads up we have a similar definition in `mira_model`'s context.py
https://github.com/DARPA-ASKEM/askem-beaker/blob/main/src/askem_beaker/contexts/mira_model/context.py#L251

Reply:
hmmm - I don't think anyone is using "mira_model" context, I am inclined to remove it albeit for a different PR.

Reply:
Makes sense
The only references to that context I see are an unused `tera-model-jupyter-panel.vue` and a "used" (only for testing at /jupyter-test `JupyterTest.vue`